### PR TITLE
Finishing up!

### PR DESCRIPTION
**Pull request by BenMueller1**

### CODE REVIEW

File: yeastdnnexplorer/probability_models/generate_data.py [hunk 1]
```diff
@@ -278,11 +284,265 @@ def default_perturbation_effect_adjustment_function(
         signal_labels[:, 0] == 1, adjusted_scores, torch.zeros_like(adjusted_scores)
     )
 
+    # NOTE: setting the noise genes to the noise mean
     adjusted_mean[signal_labels[:, 0] == 0] = noise_mean
 
     return adjusted_mean
 
 
+def default_perturbation_effect_adjustment_function(
```
Comment (cmatKhan) [inline yeastdnnexplorer/probability_models/generate_data.py:293]: update the generate_in_silico_data notebook

File: pyproject.toml [hunk 2]
```diff
@@ -15,8 +15,14 @@ pytorch-lightning = "^2.1.4"
 scikit-learn = "1.4.0"
 tensorboard = "^2.16.1"
 
+torchsummary = "^1.5.1"
```
Comment (cmatKhan) [inline pyproject.toml:18]: im going to guess that it is one of these

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ exclude: '^docs/|devcontainer.json'
 default_stages: [commit]
 
 default_language_version:
-  python: python3.10
+  python: python3.11
```
Comment (cmatKhan) [inline .pre-commit-config.yaml:5]: it is * possible * that this is a 3.11 problem, but i don't think it is

**Comment by cmatKhan** [comment]
@BenMueller1 I think this is no longer a valid pull request -- close it if so

**Comment by BenMueller1** [comment]
Closing this because we have fixed the error on another branch and will be working off of that branch from now on

**Pull request closed without merging**